transformer_engine/paddle/layer/transformer.py

```diff
@@ -139,6 +141,7 @@ def __init__(
         attention_dropout_rng_state_name: str = "local_seed",
         hidden_dropout_rng_state_name: str = "global_seed",
         backend: str = "transformer_engine",
+        use_cudagraph: bool = True,
```

Comment:
CUDA graphs are delicate, so I prefer disabling them by default:
```suggestion
        use_cudagraph: bool = False,
```
Also, this API might run into problems if a user tries to initialize some layers with `use_cudagraph=True` and other layers with `use_cudagraph=False`. The `get_global_fp8_state().enable_cudagraph()` API is somewhat better at conveying that CUDA graph support is all-or-nothing.

Reply:
You're right; let's use only `get_global_fp8_state().enable_cudagraph()` to enable cudaGraph globally.